admin_manual/configuration/user/user_auth_ldap.rst

```diff
@@ -396,6 +396,13 @@ Nested Groups:
   By default a group will only contain users, so enabling this option isn't necessary.
   However, if groups are contained inside groups, and this option is not enabled, any groups contained within other groups will be ignored and not returned in search results.
 
+  In the case you choose **Active Directory** as your LDAP server you have to enter this specific filter in the **ownCloud LDAP Server Settings** in order for the ownCloud LDAP app to be able to find users in **nested groups**.
+  
+  Example:
+
+  * ``memberof:1.2.840.113556.1.4.1941:=CN=<Group_Name>,dc=example,dc=com``
```

Comment:
I need to ask again, how do you come to the string: ``memberof:1.2.840.113556.1.4.1941``
It seems that there is a specific notation on how the complete string is beeing ceated.
Please describe that. The example alone is only 50% of the story.

Reply:
👍
yes, this is necessary.
your research helps other to fill the content properly, 
understand why and that we comply to microsoft !

Reply:
added

Reply:
great, additional question:
``CN=<Group_Name>,dc=example,dc=com``
means you have to specify the <Group_Name> with the one you want to recurse into, or ?

2 points:
if true, please write that <Group_Name> has to be replaced with the group name of choice.
what happens if you have more groups you have to recurse into. how to deal with that?
CN=<Group_Name1>, CN=<Group_Name2>, ...
or seperate lines

Reply:
I would also write that in two lines:
general string layout as you have already written
and an detailed example replacing the placeholders with content. else this is a bit confusing...